app/retail/templates/admin/index.html

```diff
@@ -71,16 +71,19 @@
 
 <div id="content-related">
     <div class="module" id="-some-custom-links">
-        <h2>{% trans 'Stats' %}</h2>
+        <h2>{% trans 'Analytics' %}</h2>
         <ul>
-            <a href="{% url 'stats' %}">Stats</a>
+            <a href="{% url 'stats' %}">KPI Dashboard</a>
         </ul>
         <ul>
             <a href="{% url 'cohort' %}">Cohort Analysis</a>
         </ul>
         <ul>
             <a href="{% url 'funnel' %}">Funnel Analysis</a>
         </ul>
+        <ul>
+            <a href="/_administration/viz/">Analytics Playground</a>
```

Comment:
When possible, we should use `url()` in the event the url path changes.

Reply:
updated